jenkins/internal-ci/generic/pr/component_test/Motr_Pre_merge_DTM_Integration.groovy

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env groovy
+// CLEANUP REQUIRED
+pipeline { 
+    agent {
+        node {
+            label "docker-${OS_VERSION}-node"
+        }
+    }
+
+    options { 
+        skipDefaultCheckout()
+        timeout(time: 180, unit: 'MINUTES')
+        timestamps()
+        disableConcurrentBuilds()
+        ansiColor('xterm')  
+    }
+
+    parameters {  
+	    string(name: 'MOTR_REPO', defaultValue: 'https://github.com/Seagate/cortx-motr', description: 'Repo for Motr')
+        string(name: 'MOTR_BRANCH', defaultValue: 'main', description: 'Branch for Motr')
+        }
+
+    environment {
+
+        // Motr Repo Info
+
+        GPR_REPO = "https://github.com/${ghprbGhRepository}"
+        MOTR_URL = "${ghprbGhRepository != null ? GPR_REPO : MOTR_URL}"
```

Comment:
rename both `MOTR_URL` variables to `MOTR_REPO`

Reply:
Done